src/lib.rs

```diff
@@ -20,6 +20,14 @@ extern crate alloc;
 #[cfg(feature = "unwinder")]
 mod unwinder;
 
+#[cfg(all(feature = "unwinder", feature = "fde-custom"))]
+pub mod custom_fde_finder {
+    pub use super::unwinder::{
+        set_custom_fde_finder, BaseAddressFinder, FDEFinder, FDESearchResult,
+        SetCustomFDEFinderError,
+    };
+}
```

Comment:
How would you like these items, now necessarily public, to be exposed?

Reply:
I don't really want to expose the internals. Maybe instead of custom "FDEFinder", it could be a custom "EhFrameFinder" (which you currently call `BaseAddressFinder`) that is registered? The whole FDE finding mechanism can still be internal.

Reply:
Done.